Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -63,6 +67,7 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     void setActiveAnnotation(RetainPtr<PDFAnnotation>&&) final;
     void focusNextAnnotation() final;
     void focusPreviousAnnotation() final;
+    RetainPtr<PDFAnnotation> getNextTextAnnotation(AnnotationSearchDirection) const;
```

Comment:
Getters in WebKit for not have a "get" prefix, per coding style.

Reply:
Fixed